<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinningServiceAddDialog</class>
 <widget class="QDialog" name="PinningServiceAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configure a new remote pinning service</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="name">
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="provider">
       <item>
        <property name="text">
         <string>Pinata</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Web3.storage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Estuary.tech</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nft.storage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QTextEdit" name="secret"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="endpoint">
       <property name="maxLength">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pinning service name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Service provider</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Paste here the remote pinning service API key (JWT format)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="endpointCustom">
       <property name="text">
        <string>Use custom endpoint</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Endpoint</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="vLayoutInfo"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>endpoint</tabstop>
  <tabstop>secret</tabstop>
  <tabstop>provider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
